Prox/Prox/MapPlacesTransition.swift

```diff
@@ -161,7 +165,7 @@ class MapPlacesTransition: NSObject, UIViewControllerAnimatedTransitioning {
             fakeImageView.contentMode = UIViewContentMode.scaleAspectFill
             fakeImageView.clipsToBounds = true
             fakeImageView.backgroundColor = .red
```

Comment:
This might be the red that you're seeing. I must have left this in from debugging :(